src/services/OrderService.ts

```diff
@@ -290,6 +290,37 @@ const retryShippingLabel = async (shipmentIds: Array<string>): Promise<any> => {
   })
 }
 
+const fetchShipmentLabelError = async (shipmentIds: Array<string>): Promise<any> => {
+  let shipmentLabelError = [];
+  const params = {
+    "entityName": "ShipmentPackageRouteSeg",
+    "inputFields": {
+      "shipmentId": shipmentIds,
+      "gatewayMessage": null,
+      "gatewayMessage_op": "not-equal",
+      "gatewayStatus": "error",
+      "gatewayStatus_op": "equals"
+    }
+  }
+
+  try {
+    const resp = await api({
+      url: "performFind",
+      method: "get",
+      params
+    })
+
+    if (!hasError(resp)) {
```

Comment:
Follow the pattern that if there is some error throw the error, otherwise perform the success operation, like here:
https://github.com/hotwax/fulfillment-pwa/pull/215/files#diff-42325f64b3fb9a1683655b01d10fe216c4adf02ba7d1838439e0da1d5c499f4dL242-L244